sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -693,4 +694,29 @@ case class HiveTableRelation(
   override def newInstance(): HiveTableRelation = copy(
     dataCols = dataCols.map(_.newInstance()),
     partitionCols = partitionCols.map(_.newInstance()))
+
+  override def simpleString(maxFields: Int): String = {
+    val catalogTable = tableMeta.storage.serde match {
+      case Some(serde) => tableMeta.identifier :: serde :: Nil
+      case _ => tableMeta.identifier :: Nil
+    }
+
+    val metadataEntries = Map(
+      "CatalogTable" -> catalogTable.mkString(", "),
```

Comment:
is it also how file source tables display table info?

Reply:
> is it also how file source tables display table info?

File source tables won't show this.
```
== Parsed Logical Plan ==
'Project [*]
+- 'Filter ('df.k < 3)
   +- 'UnresolvedRelation [df], []

== Analyzed Logical Plan ==
id: bigint, k: bigint
Project [id#7L, k#8L]
+- Filter (k#8L < cast(3 as bigint))
   +- SubqueryAlias spark_catalog.default.df
      +- Relation[id#7L,k#8L] parquet

== Optimized Logical Plan ==
Filter (isnotnull(k#8L) AND (k#8L < 3))
+- Relation[id#7L,k#8L] parquet

== Physical Plan ==
*(1) ColumnarToRow
+- FileScan parquet default.df[id#7L,k#8L] Batched: true, DataFilters: [], Format: Parquet, Location: InMemoryFileIndex[file:/Users/angerszhu/Documents/project/AngersZhu/spark/sql/hive/target/tmp/hiv..., PartitionFilters: [isnotnull(k#8L), (k#8L < 3)], PushedFilters: [], ReadSchema: struct<id:bigint>

```